src/modules/commander/HomePosition.hpp

```diff
@@ -76,6 +78,14 @@ class HomePosition
 
 	uORB::SubscriptionData<vehicle_global_position_s>	_global_position_sub{ORB_ID(vehicle_global_position)};
 	uORB::SubscriptionData<vehicle_local_position_s>	_local_position_sub{ORB_ID(vehicle_local_position)};
+	uORB::Subscription _vehicle_air_data_sub{ORB_ID(vehicle_air_data)};
+
+	uint64_t _last_gps_timestamp{0};
+	uint64_t _last_baro_timestamp{0};
+	AlphaFilter<float> lpf_baro{5.f};
```

Comment:
Maybe just put the 5 s time constant in a static constexpr just above, so any reader will immediately understand what this 5.f means.